components/button/button_constants.js

```diff
@@ -55,6 +55,14 @@ export const INVALID_COMBINATION = [
   },
 ];
 
+export const BUTTON_ICON_SIZES = {
+  xs: '200',
+  sm: '200',
+  md: '300',
+  lg: '400',
+  xl: '500',
+};
```

Comment:
When I choose an icon _before_ choosing a Button size, it's not aligned to the default icon size for the default button size.

<img width="106" alt="image" src="https://user-images.githubusercontent.com/1165933/202300615-83d90d8c-5740-40ca-b8f2-e34d88100650.png">